popdat/Cargo.toml

```diff
@@ -6,9 +6,13 @@ edition = "2018"
 
 [dependencies]
 abstutil = { path = "../abstutil" }
+geo = "0.16.0"
+geojson = { version = "0.21.0", features = ["geo-types"] }
 geom = { path = "../geom" }
 log = "0.4.11"
 map_model = { path = "../map_model" }
 rand = "0.7.0"
 rand_xorshift = "0.2.0"
 sim = { path = "../sim" }
+serde_json = "1.0.60"
```

Comment:
Please sort. I really wish cargo fmt would get support for this :(

Reply:
I don't know why this is so hard for me. Thanks for your diligence!